src/api/app/assets/stylesheets/webui/application/announcements.scss

```diff
@@ -2,6 +2,8 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
+// TODO: delete this file on final migration to bootstrap (switch_to_webui2)
```

Comment:
Please add the [# bento_only](https://github.com/openSUSE/open-build-service/wiki/How-to-work-in-Bootstrap-%28new-UI%29#things-to-do-after-the-migration) tag. That way we can easily grep through our repo once via done with the migration.

Reply:
Done!